src/components/pages/ProfilePage/components/ArrowBack/ArrowBack.module.css

```
@@ -0,0 +1,9 @@
+.arrowBack {
+  margin-top: 32px;
+  cursor: pointer;
+  transition: all 0.5s ease-in-out;
+}
+
+.arrowBack:hover svg path {
```

to apply styles when imported as ReactComponent
```suggestion
.arrowBack:hover path {
```